packages/website/src/examples/form/button/ButtonWithAsyncLoader.demo.tsx

```diff
@@ -8,6 +9,15 @@ const promise = async () => {
         title: 'Saved successfully',
         message: 'The save disabled was put in a loading state while it was waiting for the save to resolve.',
         autoClose: false,
+        icon: <CheckmarkSize24Px height={24} />,
+        styles: (theme) => ({
+            root: {
+                '.__mantine-ref-icon': {
+                    backgroundColor: 'transparent',
+                    color: theme.colors.success[6],
+                },
+            },
+        }),
```

Comment:
I'm sorry to be that guy, but having to write this each time we want to show a notification isn't acceptable IMO 😬 We will have to find another way.

I was think we could just specify `color: 'success'` here, and hide this complexity in the Theme itself

Reply:
I did the PR exactly for that purpose, don't apologize :P I will fin a way from the theme